packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapCircleController.h

```diff
@@ -19,10 +19,9 @@
 
 // Defines circle controllable by Flutter.
 @interface FLTGoogleMapCircleController : NSObject <FLTGoogleMapCircleOptionsSink>
```

Comment:
Does `FLTGoogleMapCircleOptionsSink` protocol need to exist?  It looks like it's only used in the implementation of this class, and is only implemented by `FLTGoogleMapCircleController`.  You could just put the properties directly on `FLTGoogleMapCircleController`.